### DISCUSSION TITLE
Test du Capteur Co2 Heiman HS3AQ - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#14] [Comments]
# Test du Capteur Co2 Heiman HS3AQ - Haade.fr

test du capteur de température, humidité, Co2 Heiman Zigbee et intégration dans homeassistant

https://haade.fr/fr/blog/test-capteur-co2-zigbee-heiman-HSAQ

**Comment by Lsda26** [comment]
Bonjour,
Comment lance t'on l’appairage sur cet appareil pour sa détection dans Zigbee2MQTT?

**Comment by haade-administrator** [comment]
salut c'est dans l'article: 
"L’appairage se fait en maintenant enfoncé les touches + et - durant 5 secondes, suite à ça une indication online apparaît il suffira de rappuyer sur la touche + pour activer le décompte d’une minutes pour intégrer le module dans l’assistant domotique."

**Comment by Lsda26** [comment]
Super merci. J'avais pas vu.Le 3 juin 2023 11:17, haade ***@***.***> a écrit :
salut c'est dans l'article:
"L’appairage se fait en maintenant enfoncé les touches + et - durant 5 secondes, suite à ça une indication online apparaît il suffira de rappuyer sur la touche + pour activer le décompte d’une minutes pour intégrer le module dans l’assistant domotique."

**Comment by pat12392** [comment]
Bjr
je possède ce capteur, que j'utilise aussi via jeedom, mais je rencontre un problème avec la température. Celle qui est affichée n'est pas bonne et j'ai créé un offset de 2 pour la corriger.
problème : la correction est bien prise en compte via jeedom dans l'affichage du widget associé, mais la température sur le capteur lui-même n'est pas corrigée.
Y a t-il moyen de réinjecter la valeur corrigée sur le module lui-même ?

**Comment by SebOM** [comment]
Bar,
J'ai également le capteur, pas de pb avec la température, l'humidité ou la qualité de l'air.
Par contre l'heure se décale, en effet elle perd environ 10 min toute les 24h. 
Auriez vous une solution pour mon pb.
